src/app/pages/website/about-us/about-us.component.html

```diff
@@ -1,43 +1,43 @@
-      bij scholen
-      en docenten. Denk hierbij aan wat scholen nodig hebben om een match te vinden binnen onze lijst van
-      gastdocenten.
-    </p>
-
-    <h2>Scrumteam Gastdocenten</h2>
-    <p>
-      Binnen Scrumteam Gastdocenten is Jasper Sprengers Scrum Master. Hij zorg ervoor dat zijn team informatie haalt bij
-      toekomstige gast docenten. Denk hierbij aan welke lessen zij willen geven en in welke regio.
-    </p>
-
-    <h2>Scrumteam Marketing / Marktonderzoek</h2>
-    <p>
-      Binnen Scrumteam Markerting / Marktonderzoek is Eddy Vos Scrum Master. Hij zorgt ervoor dat zijn team onderzoek doet
-      naar mogelijke partners die dit platform kunnen versterken. Denk hierbij aan sponsors die hun bedrijfsnaam graag
-      willen koppelen aan dit initiatief. Of aan andere organisaties die lesmodules verzorgen maar geen docenten hebben.
-    </p>
-
-    <h2>Alle hulp is welkom</h2>
-    <p>
-      Het streven is om in 2022 het platform te lanceren. Wanneer in 2022 is afhankelijk van de beschikbaarheid van onze
-      vrijwilligers. Mocht dit initiatief jouw interesse gewekt hebben en wil jij hieraan meehelpen, neem dan contact met
-      ons op via ons contactformulier!
-    </p>
-
-    <littil-button [inline]="true" (onClick)="navToContact()">Contact us</littil-button>
+  <p>Project LITTIL is ontstaan uit een idee van Stichting Devoxx4Kids Nederland. Deze stichting heeft
+    meerdere
+    aanvragen van scholen gehad om een gastles te komen verzorgen.
+  </p>
+  <p>
+    Wij willen deze scholen graag helpen om een gast docent te vinden in hun omgeving die aansluit op hun wensen. Daarom
+    zijn wij gestart met 3 scrum teams en 60 vrijwilligers die hier onderzoek naar doen.
+  </p>
+
+  <h2>Scrumteam Scholen</h2>
+  <p>Binnen Scrumteam Scholen is Hans Lieben Scrum Master. Hij zorgt ervoor dat zijn team informatie haalt
+    bij scholen
+    en docenten. Denk hierbij aan wat scholen nodig hebben om een match te vinden binnen onze lijst van
+    gastdocenten.
+  </p>
+
+  <h2>Scrumteam Gastdocenten</h2>
+  <p>
+    Binnen Scrumteam Gastdocenten is Jasper Sprengers Scrum Master. Hij zorg ervoor dat zijn team informatie haalt bij
+    toekomstige gast docenten. Denk hierbij aan welke lessen zij willen geven en in welke regio.
+  </p>
+
+  <h2>Scrumteam Marketing / Marktonderzoek</h2>
+  <p>
+    Binnen Scrumteam Markerting / Marktonderzoek is Eddy Vos Scrum Master. Hij zorgt ervoor dat zijn team onderzoek doet
+    naar mogelijke partners die dit platform kunnen versterken. Denk hierbij aan sponsors die hun bedrijfsnaam graag
+    willen koppelen aan dit initiatief. Of aan andere organisaties die lesmodules verzorgen maar geen docenten hebben.
+  </p>
+
+  <h2>Alle hulp is welkom</h2>
+  <p>
+    Het streven is om in 2022 het platform te lanceren. Wanneer in 2022 is afhankelijk van de beschikbaarheid van onze
+    vrijwilligers. Mocht dit initiatief jouw interesse gewekt hebben en wil jij hieraan meehelpen, neem dan contact met
```

Comment:
Deze tekst moeten we nog even aanpassen.  Staat een beetje raar als we live gaan met de melding dat we er naar streven om in 2022 live te gaan. Kunnen we zondag wel doen bij de hackaton.